src/MatchingGame.js

```diff
@@ -61,12 +65,21 @@ class MatchingGame extends React.Component {
         return images[Math.floor(Math.random()*images.length)];
     }
 
+    renderIndicator(condition) {
+        if (condition === "C" || condition === "D") {
+            return (
+                <button className="indicatorButton" onClick={() => this.sendIndicatorInteraction()}>Indicator Button for condition {this.props.condition}</button>
+            )
+        }
+    }
+
```

Comment:
Instead of having a function with logic to control when the indicator shows up in the MatchingGame class, let's instead create a new component for the indicator. Yes, this will be a very simple component that basically is just this function but it allows for better seperation of concerns between the matching game and the indicator.